src/runtime/hexagon_host.cpp

```diff
@@ -71,6 +73,20 @@ WEAK void poll_log(void *user_context) {
     }
 }
 
+WEAK int get_remote_profiler_func() {
+    if (!remote_poll_profiler_func) {
+        // If profiling isn't supported remotely, bill everything to
+        // 'overhead'
+        return 0;
```

Comment:
It'd be nice if we could just bill the Hexagon calling func instead. That was the old behavior, which was somewhat useful (told you how much time was spent in each Hexagon pipeline).